Cargo.toml

```diff
@@ -0,0 +1,4 @@
+[workspace]
+members = [
+  "crypto"
+]
```

Comment:
We'll restructure the repo to hold a single crate in the future - but this is fine for now.